src/Functions/FunctionFactory.h

```diff
@@ -82,3 +82,37 @@ class FunctionFactory : private boost::noncopyable,
 };
 
 }
+
+typedef void (*func_entry)(DB::FunctionFactory &factory);
+
+#define FUNCTION_COLD_INIT __attribute__((__cold__))
```

Comment:
The attribute isn't really related to "init(ializatio)". Maybe just "ATTR_IS_COLD" (ATTR for "attribute")?

Reply:
Thanks, renamed